docs/origin-story.rst

```diff
@@ -115,9 +115,7 @@ Instead, we had hundreds of signups for the mailing list in the first
 day. Over the next couple months we were contacted by people from a
 variety of disciplines; technical writing industry groups, designers,
 typographers, literate programmers, API driven startups, and more. We
-quickly `found a new
-venue <http://docs.writethedocs.org/conference/venue/>`__,
-and sold out a 250 person conference. The
+quickly found a new venue, and sold out a 250 person conference. The
```

Comment:
I couldn't find a link that made sense and worked, so it was suggested I remove it. Maybe it could link to https://www.mcmenamins.com/mission-theater ?  I don't think linking to the 2013 conference index is enlightening.